src/index.html

```diff
@@ -17,35 +17,35 @@
   </head>
   <body>
     <header class="header">
-      <a class="header__logo" href="">
+      <a class="header__logo" href="home">
```

Comment:
The href value for the header__logo link should start with a '/' to indicate the root directory (e.g. '/home')